next-env.d.ts

```diff
@@ -1,6 +1,3 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
 /// <reference types="next/image-types/global" />
-
-// NOTE: This file should not be edited
-// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
@resir014 do you know why these lines are automatically added (or removed) differently in different PRs? 😅

Reply:
Is this has something to do with prettier configuration on local vscode?

Reply:
@zainfathoni I think that got added on 11.0 but got removed again on 11.1 😅